src/go/rdctl/cmd/shell.go

```diff
@@ -116,21 +127,37 @@ func setupLimaHome() error {
 const restartDirective = "Either run 'rdctl start' or start the Rancher Desktop application first"
 
 func checkLimaIsRunning(commandName string) bool {
-	// Ignore error messages; none are expected here
-	output, err := exec.Command(commandName, "ls", "0", "--format", "{{.Status}}").CombinedOutput()
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+
+	cmd := exec.Command(commandName, "ls", "0", "--format", "{{.Status}}")
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	err := cmd.Start()
+	if err == nil {
+		err = cmd.Wait()
+	}
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Failed to run 'rdctl shell': %s\n", err)
 		return false
 	}
-	if strings.HasPrefix(string(output), "Running") {
+	limaState := strings.TrimRight(stdout.String(), "\n")
+	// We can do an equals check here because we should only have received the status for VM 0
+	if limaState == "Running" {
 		return true
 	}
-
-	if output == nil || strings.Contains(string(output), "No instance matching 0 found") {
+	if limaState != "" {
+		fmt.Fprintf(os.Stderr,
+			"The Rancher Desktop VM needs to be in state \"Running\" in order to execute 'rdctl shell', but it is currently in state \"%s\".\n%s.\n", limaState, restartDirective)
```

Comment:
Use `%q` to include a quoted string:

```suggestion
			"The Rancher Desktop VM needs to be in state \"Running\" in order to execute 'rdctl shell', but it is currently in state %q.\n%s.\n", limaState, restartDirective)
```